.github/workflows/test.yml

```diff
@@ -20,20 +20,29 @@ env:
 jobs:
   build:
     name: test
-    runs-on: ubuntu-20.04
     strategy:
       matrix:
         include:
           - rust: stable
             target: x86_64-unknown-linux-gnu
+            os: ubuntu-20.04
           - rust: stable
             target: aarch64-unknown-linux-musl
+            os: ubuntu-20.04
           - rust: stable
             target: x86_64-fortanix-unknown-sgx
+            os: ubuntu-20.04
           - rust: beta
             target: x86_64-unknown-linux-gnu
+            os: ubuntu-20.04
           - rust: nightly
             target: x86_64-unknown-linux-gnu
+            os: ubuntu-20.04
+          - rust: stable
+            target: x86_64-apple-darwin
+            os: macos-latest
+
+    runs-on: ${{ matrix.os }}
```

Comment:
```suggestion
```
This line is redundant!

Reply:
I don't think so. I'll give it a try ...

Reply:
Without that line, no tests at all are run.

Reply:
I see you are right ! :cry:

Reply:
no worries